charts/operator-wandb/templates/ingress.yaml

```
@@ -80,3 +81,64 @@ spec:
               number: 8082
   {{- end }}
 {{- end }}
+{{- end }}
+---
+{{- if .Values.gcpPrivatLinkIngress.create }}
```

```suggestion
{{- if .Values.ingress.secondary.create }}
```

Lets generalize and call it `secondary`